components/Navbar/NavbarMobile/TabGroupMobile.tsx

```diff
@@ -0,0 +1,64 @@
+import TabMobile from './TabMobile'
+import { chapters, lessons } from 'content'
+import { useLang, useTranslations } from 'hooks'
+
+import clsx from 'clsx'
+
+export default function TabGroup({ isOpen, clicked, params }) {
+  const { slug } = params
+
+  const lang = useLang()
+  const t = useTranslations(lang)
+
+  const chapter = chapters[slug]
+
+  if (!chapter) {
+    return null
+  }
+
+  const chapterMeta = chapter?.metadata
+
+  const challenges = chapter.metadata.challenges.map((lessonId) => {
+    let { title } = lessons[lessonId].metadata
```

Comment:
Can be `const`

Reply:
Thanks for catching this.

Addressed in 9b4c5743cb1375d2eae011ab956848b2f92682c6